conf/reference/multicollins_reference.conf

```diff
@@ -3,4 +3,5 @@ multicollins {
   instanceAssetType = DATA_CENTER
   locationAttribute = LOCATION
   thisInstance = NONE
+  queryTimeout = 30
```

Comment:
Why is this value `queryTimeout` but the config class is referencing `queryCacheTimeout`?